### DISCUSSION TITLE
We were not able to compare the content with the source, please refresh or report the issue.

### DISCUSSION THREAD

**Discussion by wrabit** [codesandbox/codesandbox-client#4430] [General]
Linked a github project to sandbox, https://codesandbox.io/s/strange-waterfall-k20vc?fontsize=14&hidenavigation=1&module=/src/App.vue&theme=dark

What do we refresh and how?

**Comment by garethx** [comment]
We deployed an update which caused a misleading message. Thanks for reporting and should be fixed now!

**Reply by ADICCFUTS1** [reply]
?????????????????????????????????????????????????????????????????????????????????????????????????????????? BUG

**Reply by JamesACS** [reply]
@ADICCFUTS1 This issue is two years old, if you're encountering a bug please create a new issue, or search for a more recent one and provide as much detail as possible so we can investigate.
